### ISSUE TITLE
Make model return logits, not softmax

### ISSUE THREAD

**Issue by petermatts** [petermatts/CarSnap-AI#22]

**Comment by petermatts** [comment]
Done
